images/server/install-packages.sh

```diff
@@ -13,18 +13,28 @@ get_custom_repo() {
 install_packages_from="$1"
 samba_version_suffix="$2"
 install_custom_repo="$3"
+
+# shellcheck disable=SC1091
+OS_BASE="$(. /etc/os-release && echo "${ID}")"
```

Comment:
We'll have to move the above `OS_BASE` calculation towards the beginning of the script(before _get_custom_repo()_). Because installation of curl is only needed for Fedora as CentOS Stream 9 images comes with _curl-minimal_ pre-installed which conflicts with original _curl_ package.

Reply:
I don't know why @obnoxxx local build didn't hit that error. But rather than that change could we do `dnf install -y /usr/bin/curl` instead? I tried it on containers of both `registry.fedoraproject.org/fedora:37` and `quay.io/centos/centos:stream9`  (by hand) and in both cases it did the right thing - nothing on centos stream9 and installed curl on fedora.

Reply:
`dnf install /usr/bin/curl` also works.

Reply:
@anoopcs9  writes:
> `dnf install /usr/bin/curl` also works.

This is what I just added to the PR ... @phlogistonjohn - FYI

Reply:
> @obnoxxx Above change is still required.

I did the `/usr/bin/curl` thing @anoopcs9